app/models/user.rb

```diff
@@ -1,4 +1,8 @@
 class User < ApplicationRecord
+  # Include default devise modules. Others available are:
+  # :confirmable, :lockable, :timeoutable, :trackable and :omniauthable
+  devise :database_authenticatable, :registerable,
+         :recoverable, :rememberable, :validatable, :confirmable
   has_many :posts, foreign_key: 'author_id'
   has_many :likes, foreign_key: 'author_id'
   has_many :comments, foreign_key: 'author_id'
```

Comment:
- [ ] Dear Team, Your functional implementations great, But I notice your layout is a bit off compared with the provided wireframe. Please keep inmind the views in devise for the registration and login is expected to be modified. 

### Your design 
![image](https://github.com/mudasiri/mybloggerapp/assets/69077061/59749c42-d229-4954-ac63-2b4e156e2118)

### The wireframe
![blog_login](https://github.com/mudasiri/mybloggerapp/assets/69077061/873bb0b5-540d-4d6d-9331-3033ee58ef60)